keras_nlp/models/falcon/falcon_mlp.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Falcon MLP"""
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras.layers import Dense
+from tensorflow.keras.activations import gelu
+from keras_nlp.api_export import keras_nlp_export
+
+
+@keras_nlp_export("keras_nlp.layers.FalconMLP")
+class FalconMLP(keras.layers.Layer):
```

Comment:
We can move this to decoder layer

Reply:
@Ruqyai update on this?
I recommend you to go through other transformers. We don't create a seperate MLP class.